templates/Gemfile.tt

```diff
@@ -46,6 +46,14 @@ gem "webpacker"
 gem "<%= @panacea.dig("background_job") %>"
 <% end -%>
 
+<% if @panacea.dig("bootswatch") %>
+# Bootswatch Dependencies
+gem "bootstrap-sass"
```

Comment:
Hey @AAlvAAro after playing a little with your branch on my local, I got an error that says:

`Could not find gem 'bootstrap-sass' in any of the gem sources listed in your Gemfile.`

After investigating a little, I think the solution is to use the following declaration:

`gem "bootstrap-sass", "~> 3.3.7"`

After that change the error disappeared.

Reply:
@efigarolam that change is done! 👀